bindings/gumjs/gumdukstream.c

```diff
@@ -666,6 +668,29 @@ GUMJS_DEFINE_FUNCTION (gumjs_output_stream_write_all)
       GUM_DUK_WRITE_ALL);
 }
 
+GUMJS_DEFINE_FUNCTION (gumjs_output_stream_write_memory_region)
+{
+  GumDukObject * self;
+  GumDukHeapPtr callback;
+  GumDukWriteOperation * op;
+  gconstpointer address;
+  gsize length;
+
+  (void) ctx;
```

Comment:
We used to treat unused arguments as warnings, but only on Windows, but I ended up disabling that as there's so many cases where it's unavoidable and we just end up with "noisy" code. For now I think we should avoid adding these to newly written code. (Would be nice to remove all of these "suppressors" in one swoop.)